third_party/npm_react-json-view-lite/LICENSE

```diff
@@ -0,0 +1 @@
+MIT License
```

Comment:
It's not really a proper license/copyright statement, but let's make this `LICENSE` file exactly what's in the upstream README under the License section:
```
MIT © AnyRoad (https://github.com/AnyRoad)
```